tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -188,6 +188,29 @@ function get_max_arc_size
 	echo $max_arc_size
 }
 
+function get_max_dbuf_cache_size
+{
+	typeset -l max_dbuf_cache_size
+	typeset -l param_file='/sys/module/zfs/parameters/dbuf_cache_max_bytes'
```

Comment:
FWIW since we have a handful of other test cases which need to get/set some system tunables we added a few [helper functions](https://github.com/zfsonlinux/zfs/blob/master/tests/zfs-tests/include/libtest.shlib#L3272..L3350).  The non-Linux case doesn't work but you could use `get_tunable dbuf_cache_max_bytes` for the Linux case.

Reply:
Thanks, updated.